.github/CONTRIBUTING.md

```diff
@@ -133,7 +136,11 @@ logically independent patches which build on each other.  This makes large
 changes easier to review and approve which speeds up the merging process.
 * Try to keep pull requests simple. Simple code with comments is much easier
 to review and approve.
-* Test cases should be provided when appropriate.
+* All proposed changes must be approved by a ZFS on Linux organization member.
+
+#### Tests and Benchmarks
+* Test cases should be provided when appropriate,
```

Comment:
This comma should be change to a period.

Reply:
Done!